docs/guide/hello-world.md

```diff
@@ -53,14 +53,12 @@ touch client/my-sites/hello-world/controller.js
 There you'll write your controller with a function called `helloWorld`:
 
 ```javascript
-const Controller = {
-	helloWorld( context, next ) {
-		console.log( 'Hello, world?' );
-		next();
-	},
+export const helloWorld = ( context, next ) => {
+	console.log( 'Hello, world?' );
+	next();
 };
 
-export default Controller;
+export default helloWorld;
```

Comment:
This default export can be removed entirely, it doesn't make much sense in this context.